src/backend/distributed/connection/remote_commands.c

```diff
@@ -419,6 +419,44 @@ ExecuteCriticalRemoteCommand(MultiConnection *connection, const char *command)
 }
 
 
+/*
+ * ExecuteRemoteCommandInConnectionList executes a remote command, on all connections
+ * given in the list, that is critical to the transaction. If the command fails then
+ * the transaction aborts.
+ */
+void
+ExecuteRemoteCommandInConnectionList(List *nodeConnectionList, const char *command)
```

Comment:
Could use `ExecuteCriticalRemoteCommand`

Reply:
You meant, for each connection in the list call ExecuteCriticalRemoteCommand()?. I thought so, but that will serialize the send-process/send-process....
In the cluster clock scenario, I was trying to avoid the delay and send get_cluster_clock() asap and process result later
send/send.. and ...process/process.

It's not really a big deal, I'm fine either way.

Reply:
Ah, sorry, I think you are right, it is better to parallelize